data/development/migrations/20191217125931_createTables.js

```diff
@@ -19,7 +19,7 @@ exports.up = function(knex) {
       .notNullable()
       .references('id')
       .inTable('categories')
-      tbl.integer('price')
+      tbl.float('price')
```

Comment:
Here is a post on stack overflow about why using float is usually not recommended for keys. [Link](https://stackoverflow.com/questions/10737004/should-we-use-float-as-primary-key-in-sql-server )

Read over that and let me know what you think

Reply:
Price is not a primary key, it is meant to be an integer with a decimal.

Reply:
You're right, I totally misread that. 😂Guess I need more coffee before I keep working.